docs/remote/containers-advanced.md

```diff
@@ -162,43 +202,56 @@ To connect to both:
+## Developing inside a container on a remote Docker host
 
-Occasionally you may want to use the Remote - Containers extension to develop inside a container that sits on remote server. While we are looking at ways to optimize this experience, this section will outline how you can achieve this today by attaching to a remote container or using Docker Compose.
+Occasionally you may want to use the Remote - Containers extension to develop inside a container that sits on remote server. While we are looking at ways to optimize this experience, this section will outline how you can achieve this today by attaching to a remote container from VS Code or using Docker Compose and `devcontainer.json`.
 
-### Accessing Docker Remotely
+### Using SSH to connect to remote Docker host
 
-If you have a remote [Docker Machine](https://docs.docker.com/machine/overview/) running on a remote server, you can work with it from your local machine using the Docker CLI by specifying [environment variables like `DOCKER_HOST`, `DOCKER_CERT_PATH`, `DOCKER_TLS_VERIFY`](https://docs.docker.com/machine/reference/env/). However, this port is often not exposed publicly since it can leave the machine vulnerable if not secured properly. (For example, installing Docker CE / Desktop does not open up a port by default.)
+You can use the Docker CLI locally with a remote [Docker Machine](https://docs.docker.com/machine/overview/) by setting [local environment variables like `DOCKER_HOST`, `DOCKER_CERT_PATH`, `DOCKER_TLS_VERIFY`](https://docs.docker.com/machine/reference/env/). Since VS Code uses the Docker CLI under the hood, you can use these same environment variables to connect the Remote - Containers extension to this same remote host.
 
-A more secure way is to use an SSH tunnel to access the Docker socket as needed. If you have an [OpenSSH compatible SSH client](/docs/remote/troubleshooting.md#installing-a-supported-ssh-client) installed, you can run the following commands in a local terminal / command prompt to start up VS Code so that it connects to the remote SSH host. Replace `user@hostname` with the appropriate remote user and hostname / IP for your server and ensure `code-insiders` is in your path.
+```bash
+export DOCKER_HOST=your-remote-machine-fqdn-or-ip-here:2375
+# Or on Windows: SET DOCKER_HOST=your-remote-machine-fqdn-or-ip-here:2375
+code-insiders
+```
 
-On **macOS or Linux**:
+Once set, you can use VS Code to [attach to any running container](/docs/remote/containers.md#attaching-to-running-containers) on the remote host or [use specialized, local `devcontainer.json` files to create / connect to a remote dev container](#using-devcontainerjson-to-work-with-a-remote-dev-container).
+
+However, Docker CE / Desktop will not expose the required Docker daemon TCP port by default since it can leave the machine vulnerable if not secured properly. The Docker CLI uses a local Unix socket (or named pipe on Windows) to communicate instead.
```

Comment:
Maybe start with "If you are not using Docker Machine" or something similar to make clear that these are two alternative approaches. Or split in two sections. (You probably know that Docker Machine takes care of encryption too, so the two are true alternatives.)

Reply:
YEah the issue is that even if its setup, the TCP port can be blocked.  Let me make that clearer and layer in that.